test/JDBC/input/BABEL-4279-vu-prepare.mix

```diff
@@ -0,0 +1,39 @@
+-- tsql
+CREATE TABLE test_babel_4279_t1([ABC.nfds] INT, [DEf.j] INT);
```

Comment:
Let's add tests for names with `"` as quotes.

Reply:
Resolved.